packages/filebrowser-extension/schema/browser.json

```diff
@@ -265,6 +265,20 @@
       "title": "Allow file uploads",
       "description": "Whether to allow uploading files",
       "default": true
+    },
+    "breadcrumbsLeftItems": {
+      "type": "number",
+      "title": "Breadcrumbs left items",
+      "description": "Number of directory items to show on the left side of the ellipsis in breadcrumbs",
```

Comment:
I think it would be good to have this configurable _but_ also in future to allow showing more items if there is space. Maybe we could change the settings to be "minimumLeftItems" (and in current implementation the actual value will always be the minimum.